src/main/java/uk/gov/hmcts/probate/service/tasklist/StoppedTaskListRenderer.java

```diff
@@ -23,6 +23,6 @@ protected String renderBody(CaseDetails details) {
                 .replaceFirst("<stopDate>", stoppedDate == null ? "Unknown" : stoppedDate.format(dateFormat))
                 .replaceFirst("<caseStopReasonsList>",
                         UnorderedListRenderer.render(CaseStoppedHtmlTemplate.CASE_STOP_REASONS))
-                .replaceFirst("<numWeeks>", "4");
+                .replaceFirst("<numWeeks>", "16");
```

Comment:
I dont suppose its worth you guys paramterising this numWeeks? so you get it off app.yml or env attr?
YOu've prob already been through that thought process??!??

Reply:
done, read the <numWeeks> from the applicaton.yml